plugins/inventory/opennebula.py

```diff
@@ -199,6 +200,11 @@ def _retrieve_servers(self, label_filter=None):
                     continue
 
             server['name'] = vm.NAME
+            server['id'] = vm.ID
+            if hasattr(vm.HISTORY_RECORDS, 'HISTORY'):
+                if isinstance(vm.HISTORY_RECORDS.HISTORY, Sequence) and len(vm.HISTORY_RECORDS.HISTORY) > 0:
```

Comment:
I don't see a point in checking for these as according to its XSD scheme (https://github.com/privazio/pyone/blob/master/pyone/xsd/vm.xsd#L70) if it is present then there will **always** be HOSTNAME attribute